### PR TITLE
ci(mergify): run `master` CI on `mergify/*` instead to fail early

### PR DESCRIPTION
**Pull request by michaelfig**
## Description

Run tests on `mergify/*` temporary branch so that failures prevent merging to master.

### CODE REVIEW

File: .github/workflows/after-merge.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #  Require branches to be up to date before merging
 on:
   push:
-    branches: [master] # $default-branch
+    branches: [merge/*]
```
Comment (mhofman) [inline .github/workflows/after-merge.yml]:
```suggestion
    branches: [mergify/*]
```

**Review by mhofman — CHANGES_REQUESTED** [review]
Should we keep `master` as well for the cases where mergify is bypassed? I guess the problem is if mergify doesn't do a fast forward?

**Comment by michaelfig** [comment]
> Should we keep `master` as well for the cases where mergify is bypassed? I guess the problem is if mergify doesn't do a fast forward?

If we do that, the CI will run twice.

**Review by mhofman — APPROVED** [review]

**Comment by mhofman** [comment]
LGTM once the formatting is fixed

**Comment by mhofman** [comment]
@michaelfig I can't see that any of the jobs ran on this PR before merging. Any idea why?

**Pull request merged**